### PR TITLE
Final review of therapeutics section 

### PR DESCRIPTION
**Pull request by ajlee21**
### Description of the proposed additions or changes

Did one final read through of the therapeutics manuscript and it looks good to me. Just a few very minor text changes. 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by cbrueffer** [comment]
Looks good! I found one additional instance of `Veklury^R` that should be `Veklury`, could you include that in this PR?

**Comment by agitter** [comment]
> I found one additional instance of Veklury^R that should be Veklury, could you include that in this PR?

@cbrueffer I edited that line in my proofreading.  Does this fix what you're referring to?  https://github.com/greenelab/covid19-review/pull/854/files#diff-373f1f2c340f3947a7d3a86b97c53e3d79435ff9ffe6ee537116b1261e5e0f44R221

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -111,10 +111,11 @@ Small molecule drugs today encompass a wide range of therapeutics beyond antibio
 
 Antiviral drugs against SARS-CoV-2 are designed to inhibit replication of a virus within an epithelial host cell.
 This process requires inhibiting the replication cycle of a virus by disrupting one of six fundamental steps [@isbn:978-1405136457].
-In the first of these steps, the virus attaches to the host cell, which it next penetrates through endocytosis.
+In the first of these steps, the virus attaches and enters the host cell through endocytosis.
```
Comment (rando2) [inline content/20.pharmaceuticals.md:114]:
```suggestion
In the first of these steps, the virus attaches to and enters the host cell through endocytosis.
```

**Review by rando2 — APPROVED** [review]
Looks good @ajlee21!

**Comment by cbrueffer** [comment]
@agitter yes, thanks! Didn't seem worth opening a new PR for that. Also good call on the &reg; , I assumed it was a control character that slipped in.

**Comment by rando2** [comment]
@byrdjb also suggested (over email): "Consider skipping the remdesivir brand name & making any generic names lowercase." This might be a good suggestion because I'm not sure all authors included the brand names?

**Comment by agitter** [comment]
I would be okay removing the Veklury brand name.  I agree that brand names aren't mentioned for most of the other drugs discussed here.

**Comment by rando2** [comment]
Awesome, I removed it! I realized that was there partially as kind of a "conflict of interest" flag for the trial run by Gilead, so hopefully just mentioning that they manufacture remdesivir is sufficient.

@ajlee21 Please feel free to merge or just let me know if you're happy with everything and I can do it!

**Comment by ajlee21** [comment]
Looks good to me, once checks are complete I can merge it

**Pull request merged**